### PR TITLE
ign -> gz Migrate Ignition Headers : gz-plugin

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/784

### CODE REVIEW

**Comment by scpeters** [comment]
I've opened #105 with a special branch name to test CI on macOS with https://github.com/gazebosim/gz-cmake/pull/303

**Comment by scpeters** [comment]
@osrf-jenkins run tests please

**Comment by methylDragon** [comment]
This should be good to (rebase) merge!

File: core/include/ignition/plugin/config.hh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+/* Version number */
+// #define IGNITION_PLUGIN_MAJOR_VERSION GZ_PLUGIN_MAJOR_VERSION
+// #define IGNITION_PLUGIN_MINOR_VERSION GZ_PLUGIN_MINOR_VERSION
+// #define IGNITION_PLUGIN_PATCH_VERSION GZ_PLUGIN_PATCH_VERSION
+
+// #define IGNITION_PLUGIN_VERSION GZ_PLUGIN_VERSION
+// #define IGNITION_PLUGIN_VERSION_FULL GZ_PLUGIN_VERSION_FULL
+
+// #define IGNITION_PLUGIN_VERSION_HEADER GZ_PLUGIN_VERSION_HEADER
```
Comment (chapulina) [inline core/include/ignition/plugin/config.hh:31]: We could remove these comments

**Review by chapulina — APPROVED** [review]
LGTM!

File: core/include/ignition/plugin/config.hh (hunk 1 above)
Comment (methylDragon) [reply core/include/ignition/plugin/config.hh:31]: I'll probably leave them in for convenience if we ever end up tick-tocking the macros

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**